packages/design-system/src/components/small-icon-button.stories.tsx

```diff
@@ -13,6 +13,8 @@ const iconsMap = {
   "<TrashIcon>": <TrashIcon />,
 } as const;
 
+const states = [undefined, ...smallIconButtonStates];
```

Comment:
that undefined looks weird

Reply:
I don't want support useless states in types. Having that they used for storybook only